examples/jsm/renderers/webgpu/nodes/WGSLNodeBuilder.js

```diff
@@ -61,7 +61,8 @@ const wgslMethods = {
 	dFdy: '- dpdy',
 	mod: 'threejs_mod',
 	lessThanEqual: 'threejs_lessThanEqual',
-	inversesqrt: 'inverseSqrt'
+	inversesqrt: 'inverseSqrt',
+	textureDimensions: 'textureDimensions'
```

Comment:
Is this line needed? I think `NodeBuilder.getMethod` takes the name as default if not presented in `wgslMethods`.

Reply:
you are right, removed